ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -10,23 +10,27 @@ use std::{collections::HashMap, ffi::CString};
 const TRACE_FUNC_CTRLP_ARGIDX: u16 = 0;
 
 /// A TraceBuilder frame. Keeps track of inlined calls and stores information about the last
-/// processed stackmap, call instruction and its arguments.
+/// processed safepoint, call instruction and its arguments.
```

Comment:
This comment isn't entirely correct anymore. There's no "last processed" stackmap anymore since it's part of the call now. But I'll probably rewrite this whole thing anyway soon, so there's no need to change it now.